src/gui/mzroll/background_peaks_update.cpp

```diff
@@ -319,6 +319,8 @@ void BackgroundPeakUpdate::alignWithObiWarp(){
         Q_EMIT(updateProgressBar("Aligning Samples", 0, 0));
```

Comment:
Not sure if we need this. You could display 1% to indicate that some process has started before the boost signals start working. But the oscillating progress bar is not ideal.